src/main/java/net/openhft/chronicle/core/io/UnsafeText.java

```diff
@@ -249,7 +249,11 @@ private static long appendText(long address, String s) {
         return address;
     }
 
-    private static double asDouble(long value, int exp, boolean negative, int deci) {
+    /**
+     * @deprecated Use {@link Maths#asDouble(long, int, boolean, int)} instead
+     */
+    @Deprecated(/* To be moved into Maths.asDouble in x.25 */)
+    public static double asDouble(long value, int exp, boolean negative, int deci) {
```

Comment:
Just expose so we can keep the history for now, delegate from Maths.asDouble (with additional zero-cost assertions)